automation-tests/test/e2etemp/TempProject/.eslintignore

```diff
@@ -0,0 +1,16 @@
+**/lwc/**/*.css
```

Comment:
So just a thought here (and could be a follow up) but do we want to commit TempProject to the repo or generate a new project at the beginning of the test run.  The upside of creating the project is we get test failures if something in the project changes vs being locked into a committed project in the repo.

Reply:
When I started the epic, my original intent was for TempProject to have a project with pre-generated and configured files (like Apex classes, etc...).  At the moment, none of the e2e suites use TempProject, and instead create a new project using `SFDX: Create Project`.  Although we may decide to go this route and use a project with some pre-built files, it's. not being used at the moment and so I deleted TempProject.